CMakeLists.txt

```diff
@@ -69,10 +64,18 @@ function(trusted_firmware_build)
 
 
   set(VENEERS_FILE ${TFM_BINARY_DIR}/secure_fw/s_veneers.o)
-  set(${TFM_OUT_VENEERS_FILE} ${VENEERS_FILE} PARENT_SCOPE)
-  set(${TFM_OUT_INCLUDE_PATH} ${TFM_BINARY_DIR}/install/export/tfm/include PARENT_SCOPE)
-
   set(PSA_API_NS_PATH ${TFM_BINARY_DIR}/interface/libpsa_api_ns.a)
+  set(TFM_GENERATED_INCLUDES ${TFM_BINARY_DIR}/generated/interface/include)
+  set(SID_H_FILE_PATH ${TFM_GENERATED_INCLUDES}/psa_manifest/sid.h)
+
+  set(BL2_HEX_FILE ${CMAKE_BINARY_DIR}/tfm/bin/bl2.hex)
+
+  set(BUILD_BYPRODUCTS
+    ${VENEERS_FILE}
+    ${PSA_API_NS_PATH}
+    ${BL2_HEX_FILE}
+    ${SID_H_FILE_PATH}
```

Comment:
Minor.
`SID_H_FILE_PATH` is only used here, so there is not a big value in creating a variable for it.

Reply:
Done